src/sub_movie_generator.py

```diff
@@ -0,0 +1,26 @@
+from pathlib import Path
+from typing import List, Optional
+from foxlator_lib.video import Video
+from foxlator_lib.stt.whisper_stt import WhisperSTT, AudioTextSegment
+
+from src.third_party.ffmpeg import check_ffmpeg_installed
+from src.third_party.imagemagick import check_imagemagick_installed
+
+
+class SubMovieGenerator:
+
+    def __init__(self, movie_path: Path, model_size: str, lang: Optional[str] = None):
+        if not check_ffmpeg_installed() or not check_imagemagick_installed():
+            exit()
+        self.video = Video(movie_path)
```

Comment:
Some variables are underscored whether some aren't. What is the convention?

Reply:
Yup, I get this but what's with the not underlined variables? Is video accessed outside of the class?

Reply:
I thought that there might be a situation that these values will be used someday. I can set all of them to private if you would like to

Reply:
If you think so let's let them as it is.